server/bleep/src/ee.rs

```diff
@@ -0,0 +1 @@
+pub(crate) mod webserver;
```

Comment:
Can we rename the `ee` module to something less opaque?

Reply:
`ee` is a bit of an industry standard. See: 
- https://github.com/windmill-labs/windmill/blob/main/backend/src/ee.rs
- https://github.com/strapi/strapi/tree/main/packages/core/admin/ee